metric/system/process/process_windows.go

```
@@ -268,8 +265,12 @@ func getProcName(pid int) (string, error) {
 	}()
 
 	filename, err := windows.GetProcessImageFileName(handle)
+
+	//nolint:nilerr // safe to ignore this error
 	if err != nil {
-		return "", fmt.Errorf("GetProcessImageFileName failed for pid=%v: %w", pid, err)
+		// if we're able to open the handle but GetProcessImageFileName fails then it most probably means
+		// that the process doesn't have any executable associated with it.
```

Could we detect this based on the error?  I'd rather not swallow the error if it is possible it might contain useful information, especially when it isn't the common case.

---

Yes. We can ignore the error if it's Access Denied or Operation not Permitted. I'll update the PR.